omnigan/trainer.py

```diff
@@ -379,10 +393,14 @@ def run_epoch(self):
             self.log_step_time(step_time)
 
         for d in self.opts.domains:
```

Comment:
Why do you call model.eval() before every log and model.train() right after, instead of just calling model.eval() once before the for loop and calling model.train() once after the if ? I guess there's a reason, just wondering what it is :)